packages/plugin-sequelize/src/utils/public.ts

```diff
@@ -98,13 +98,19 @@ export function getQueryInclude(info: GraphQLResolveInfo) {
 
 export function getQueryIncludeOf(
   info: GraphQLResolveInfo,
-  targetType: ValidGraphqlType,
+  target:
+    | ValidGraphqlType
+    | ((details: UntilHandlerDetails<DefaultResolverIncludeOptions>) => boolean),
```

Comment:
It is now possible to pass a match function as `target` argument instead of a type name. We can than match the target based on any property of the `until` hook.